test/CassandraConnector.js

```diff
@@ -1,25 +1,28 @@
-const cassandra = require('cassandra-driver');
+'use strict';
 const cassandraConnector = require('../src/connectors/CassandraConnector');
 const chai = require('chai');
 const expect = require('chai').expect;
 const chaiAsPromised = require('chai-as-promised');
 chai.use(chaiAsPromised); //TODO: see if you can move these test config to a new file see mocha docs
 
-const CASSANDRA_CONTACT_POINTS = process.env.CASSANDRA_CONTACT_POINTS;
-
 describe('Client', function() {
 
-  describe('#closeClient(client)', function() {
-
-    it('should resolve the promise with the client', function() {
-      const options = {
-        contactPoints: [CASSANDRA_CONTACT_POINTS],
-        keyspace: process.env.CASSANDRA_KEYSPACE
-      };
-      const client = new cassandra.Client(options);
-      return (expect(cassandraConnector.closeClient(client)).to.eventually.be.fulfilled);
+  describe('#executeQueries(queries, eventIdentifier)', function() {
+    
+    it('should resolve the promise', function() {
+      let queries = [
+        {
+          query: 'INSERT INTO Topics (id, topic, value) VALUES (?, ?, ?)',
```

Comment:
there is no table called `Topics` https://github.com/CatalystCode/fortisdeploy/blob/119d3d847826b2cbd45791b3cf41c2ce1d60144d/ops/storage-ddls/cassandra-setup.cql